### ISSUE TITLE
Is there a way to read a TW5 offline on an iPhone and Android Mobile Phone?

### ISSUE THREAD

**Issue by gesundelehre** [Jermolene/TiddlyWiki5#2763]
I already know about the TWEdit App for the iPhone, however the App is not free though.

Is there a free alternative?
Note, i only speak about reading the TW5 offline on a mobile device and not editing it.

What is the best solution to do such? Would the easiest way be, to export the whole TW5 into a pdf or export all tiddlers onto one single huge static page?

Thanks for your help

**Comment by Jermolene** [comment]
Hi @gesundelehre since this question isn't about the code or docs, it's probably better asked in the discussion forums: http://groups.google.com/group/TiddlyWiki

Anyhow, I use the iOS Dropbox app to view TiddlyWiki HTML files offline. I've also had some success with Mobile Safari's built-in reading list, which can cache TiddlyWiki's well.

**Comment by pmario** [comment]
>iOS Dropbox app to view TiddlyWiki HTML files offline

Isn't this feature discontinued? .. Pro account?

**Comment by Jermolene** [comment]
@pmario no, Dropbox have restricted their web hosting features, but the ability to access your own files is unaffected.

**Comment by Tikkoneus** [comment]
I actually got this working and can post a more thorough answer on the Google Group if you post a question there. But the short answer is that I used the DriveSync app (3rd party app which syncs a Google Drive folder to a local folder on your phone) to keep TW up to date on my phone. Then I just used Firefox + tiddlyfox plugin to view and save it. Pretty nice, too, since DriveSync can monitor file changes. Hitting save in TW saves to the phone which immediately syncs to drive which almost immediately syncs to the laptop; and vice versa.
